samples/gunzip_nx.c

```diff
@@ -58,8 +58,9 @@
 #include <errno.h>
 #include <signal.h>
 #include "nxu.h"
-#include "nx.h"
+/* #include "nx.h" */
```

Comment:
This comment could be removed from all sample files.

Reply:
Done.